plugin.py

```diff
@@ -723,6 +723,10 @@ def _do_karma(self, irc, channel, agent, recip, line, explicit=False):
         # Extract 'puiterwijk' out of 'have a cookie puiterwijk++'
         recip = recip.strip().split()[-1]
 
+        # Exclude 'c++', 'g++' or 'i++' (c,g,i), issue #30
+        if len(recip) == 1 and recip.lower() in ['c','g','i']:
```

Comment:
@pypingou 
> If the first part len(recip) == 1 needed?
I think it is (maybe I'm wrong), we need to be sure that the recipient/user/nick is exactly one letter long, and those letters are 'c','g' or 'i'?